README.md

```diff
@@ -60,6 +66,6 @@ We're building the project with [React](https://facebook.github.io/react/docs/re
 
 ### Git
 
-Aren't familiar with Git?  Spend ~15 minutes learning with this [interactive Git tutorial](https://try.github.io/levels/1/challenges/1). Note that we'd prefer you don't use `-f` or `--force` if possible, especially with pull requests.
+Aren't familiar with Git?  Spend ~15 minutes learning with this [interactive Git tutorial](https://try.github.io/levels/1/challenges/1). Note that we'd prefer you don't use `-f` or `--force` if possible, especially with a branch that you have used to create a pull request.
```

Comment:
Suggested change:
> ~Note that~ We'd prefer you don't use `-f` or `--force` ~if possible~, especially with a branch that you have used to create a pull request.

Reply:
Now that I think of it, this probably needs to be stronger than a preference, but good simplification. It also really only matters (to us) for PRs, even though it's something to be careful about in general. How about:

> Don't use `-f` or `--force`, though, with branches that you have use to create a pull request.

Reply:
Omit the "though" and I think we're good!